Fix invalid conversion from xmlError

src/obconf-qt.cpp: In function 'int main(int, char**)':
src/obconf-qt.cpp:233:36: error: invalid conversion from 'const xmlError*' {aka 'const _xmlError*'} to 'xmlErrorPtr' {aka '_xmlError*'} [-fpermissive]

Upstream-Status: Backport[https://github.com/lxqt/obconf-qt/commit/9fc5e566ea6559c27238c9dd5ae914b767f6c251]

Signed-off-by: Rob Woolley <rob.woolley@windriver.com>

From 9fc5e566ea6559c27238c9dd5ae914b767f6c251 Mon Sep 17 00:00:00 2001
From: Tsu Jan <tsujan2000@gmail.com>
Date: Sun, 14 Jan 2024 14:28:24 +0330
Subject: [PATCH] Fixed compilation

Fixes https://github.com/lxqt/obconf-qt/issues/224
---
 CHANGELOG         | 4 ++++
 CMakeLists.txt    | 2 +-
 src/obconf-qt.cpp | 2 +-
 3 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/src/obconf-qt.cpp b/src/obconf-qt.cpp
index e07c236..0d5b442 100644
--- a/src/obconf-qt.cpp
+++ b/src/obconf-qt.cpp
@@ -230,7 +230,7 @@ int main(int argc, char** argv) {
 
   /* look for parsing errors */
   {
-    xmlErrorPtr e = xmlGetLastError();
+    const xmlError *e = xmlGetLastError();
 
     if(e) {
       QString message = QObject::tr("Error while parsing the Openbox configuration file.  Your configuration file is not valid XML.\n\nMessage: %1")
